river/Cursor.zig

```diff
@@ -1095,3 +1099,44 @@ fn passthrough(self: *Self, time: u32) void {
         self.clearFocus();
     }
 }
+
+pub fn warp(self: *Self) void {
+    self.needWarp = false;
+    if (self.seat.focused_output == &server.root.noop_output) return;
+    // Warp pointer to center of the focused view/output (In layout coordinates) if enabled.
+    var layout_box: wlr.Box = undefined;
```

Comment:
`output_layout_box` would be a bit more descriptive if you don't mind the verbosity.